devnets/startNetwork.ts

```diff
@@ -288,3 +292,23 @@ function addAuthorities(genesisConfig: GenesisConfig, count: number) {
     ])
   )
 }
+
+const hrmpChannelMaxCapacity = 8
+const hrmpChannelMaxMessageSize = 512
+function addXcmHrmpChannels(
+  genesisConfig: GenesisConfig,
+  paraIds: number[],
+) {
+  genesisConfig.hrmp ??= { preopenHrmpChannels: [] }
+  for (const senderParaId of paraIds) {
+    for (const recipientParaId of paraIds) {
+      if (senderParaId === recipientParaId) continue
+      genesisConfig.hrmp.preopenHrmpChannels.push([
```

Comment:
Do hrmp channels need to be opened for both directions? I.e. 1 -> 2 and 2 -> 1?

Reply:
yes